packages/klevu-ui/src/components/klevu-product-query-popup/klevu-product-query-popup.tsx

```diff
@@ -288,7 +293,7 @@ export class KlevuProductQueryPopup {
           offset={this.popupOffset}
           elevation={3}
           useBackground={this.useBackground}
-          popupWidth={520}
+          popupWidth={this.popupWidth}
```

Comment:
why cant width also be driven from css? is it a requirement for the alignment and placement logic?